scripts/Connector/Connector_Hirose/conn_hirose_df11_tht_top.py

```diff
@@ -0,0 +1,234 @@
+#!/usr/bin/env python3
+
+import sys
+import os
+#sys.path.append(os.path.join(sys.path[0],"..","..","kicad_mod")) # load kicad_mod path
+
+# export PYTHONPATH="${PYTHONPATH}<path to kicad-footprint-generator directory>"
+sys.path.append(os.path.join(sys.path[0], "..", "..", ".."))  # load parent path of KicadModTree
+from math import sqrt
+import argparse
+import yaml
+from helpers import *
+from KicadModTree import *
+
+sys.path.append(os.path.join(sys.path[0], "..", "..", "tools"))  # load parent path of tools
+from footprint_text_fields import addTextFields
+
+series = 'DF11'
+series_long = 'DF11 through hole'
```

Comment:
It's a little annoying that `DF11` is in two places here. The script could be smarter.